tests/pawn-shop.ts

```diff
@@ -159,25 +162,86 @@ describe("PawnHub", () => {
       LENDER_KEYPAIR.publicKey
     );
     await pawnMint.mintTo(borrowerPawnTokenAccount, BORROWER_KEYPAIR, [], 1);
+
+    const tx = new Transaction();
+    // those address derivation are no longer in the package, why?
+    const [metadata] = await PublicKey.findProgramAddress(
```

Comment:
did we add "create metadata account" and "create master's edition" because we need that to freeze a token?

Reply:
Yes, the freeze feature requires the edition and in our case the master edition because we have a "unique" NFT.
So in a sense the program is more dependent on metaplex now while it wasn't at all before